### DISCUSSION TITLE
API to get commits, branches & pull requests

### DISCUSSION THREAD

**Discussion by rmani07** [community/community#27810] [Apps, API and Webhooks]
Hello,

We need help in finding API to get commits, branches & pull request using GITHub api with Jira Issue key as input, is this possible?

We want to pull information without repositories name & organization name but just using Jira Issue Key.

any help is much appreciated 

thanks

**Comment by mpboom** [comment]
Sure, there are endpoints for that:

- [Commits](https://docs.github.com/en/rest/commits/commits)
- [Branches](https://docs.github.com/en/rest/branches/branches)
- and [Pull requests](https://docs.github.com/en/rest/pulls/pulls)
